src/OnboardingSPA/components/MiniPreview/index.js

```diff
@@ -23,7 +24,6 @@ const MiniPreview = ( {
 	const content = getContents( brandUrl );
 	const titlePreview = title === '' ? content.defaultTitle : title;
 	const descPreview = desc === '' ? content.defaultDesc : desc;
-	const urlPreview = title === '' ? content.defaultUrl : titleToUrl();
```

Comment:
Is it safe to remove `content.defaultUrl` from `src/OnboardingSPA/components/MiniPreview/contents.js` as well?